nix/haskell-project.nix

```diff
@@ -62,6 +75,19 @@ in
               ++ builtins.attrValues (config.devShell.extraLibraries p);
           };
       });
+      exes =
+        (name: value:
+          builtins.foldl' (x: y: x // y) { }
+            (builtins.map
+              (x: {
+                "${x}" = {
+                  type = "app";
```

Comment:
I think the `type` here is redundant. `"${x}".program = ...` should work AFAIU flake-parts.